alarm_req_listener.cpp

```diff
@@ -123,11 +126,25 @@ bool AlarmReqListener::zmq_init_sck()
     return false;
   }
 
-  std::stringstream ss;
-  ss << "tcp://127.0.0.1:" << ZMQ_PORT; 
+  std::string sf = std::string("/var/run/clearwater/alarms");
+  std::string ss = std::string("ipc://" + sf);
```

Comment:
The original variable was called `ss` because it was a `stringstream`. This is no longer the case. I would also prefer slightly more descriptive names, maybe `sck_dir` and `sck_url`?